### PR TITLE
removed add service call, preinstl script; added order, readonly , minimum rate property and other fixes

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: python/foglamp/plugins/south/openweathermap/openweathermap.py [hunk 1]
```diff
@@ -24,42 +27,54 @@
...
-        'default': 'London'
+        'default': 'London',
+        'order': '3'
     },
     'appid': {
         'description': 'Application ID registered with OpenWeatherMap',
         'type': 'string',
-        'default': 'bbafe18fb275ae5b200d094e36c574ff'
+        'default': 'bbafe18fb275ae5b200d094e36c574ff',
+        'order': '2'
```
Comment (ashish-jabble) [inline python/foglamp/plugins/south/openweathermap/openweathermap.py]: assetName should be configurable item

File: python/foglamp/plugins/south/openweathermap/openweathermap.py [hunk 2]
```diff
@@ -71,44 +90,69 @@ def plugin_init(config):
     city = config['city']['value']
     appid = config['appid']['value']
     rate = config['rate']['value']
+    asset_name = config['asset_name']['value']
```
Comment (ashish-jabble) [inline python/foglamp/plugins/south/openweathermap/openweathermap.py]: assetName

Comment (amarendra-dianomic) [inline python/foglamp/plugins/south/openweathermap/openweathermap.py]: should be asset_name = config['assetName']['value']

File: python/foglamp/plugins/south/openweathermap/openweathermap.py [hunk 3]
```diff
@@ -134,12 +178,12 @@ def stop(self):
...
-                asset = 'OpenWeatherMap'
-                timestamp = str(datetime.now(tz=timezone.utc))
                 data = json.loads(res)
-                readings = {'wind_deg': data['wind']['deg'], 'wind_speed': data['wind']['speed'],
-                            'temperature': data['main']['temp'],
-                            'pressure': data['main']['pressure'],
-                            'visibility': data['visibility']}
-                key = str(uuid.uuid4())
-                await Ingest.add_readings(asset=asset, timestamp=timestamp, key=key, readings=readings)
+                readings = { 'wind_deg': data['wind']['deg'],
```
Comment (amarendra-dianomic) [inline python/foglamp/plugins/south/openweathermap/openweathermap.py]: data['wind']['deg'] was not not found, hence it should be removed now.

**Pull request merged**